COMET.Web.Common/Components/Selectors/DomainOfExpertiseSelector.razor.cs

```diff
@@ -0,0 +1,61 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+//  <copyright file="DomainOfExpertiseSelector.razor.cs" company="Starion Group S.A.">
+//     Copyright (c) 2024 Starion Group S.A.
+// 
+//     Authors: Sam Gerené, Alex Vorobiev, Alexander van Delft, Jaime Bernar, Théate Antoine, João Rua
+// 
+//     This file is part of COMET WEB Community Edition
+//     The COMET WEB Community Edition is the Starion Group Web Application implementation of ECSS-E-TM-10-25 Annex A and Annex C.
+// 
+//     The COMET WEB Community Edition is free software; you can redistribute it and/or
+//     modify it under the terms of the GNU Affero General Public
+//     License as published by the Free Software Foundation; either
+//     version 3 of the License, or (at your option) any later version.
+// 
+//     The COMET WEB Community Edition is distributed in the hope that it will be useful,
+//     but WITHOUT ANY WARRANTY; without even the implied warranty of
+//     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//    Affero General Public License for more details.
+// 
+//    You should have received a copy of the GNU Affero General Public License
+//    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//  </copyright>
+//  --------------------------------------------------------------------------------------------------------------------
+
+namespace COMET.Web.Common.Components.Selectors
+{
+    using CDP4Common.SiteDirectoryData;
+
+    using COMET.Web.Common.ViewModels.Components.Selectors;
+
+    using Microsoft.AspNetCore.Components;
+
+    /// <summary>
+    /// Component used to select a <see cref="DomainOfExpertise" />
+    /// </summary>
+    public partial class DomainOfExpertiseSelector
+    {
+		/// <summary>
```

Comment:
Identation issue in the file

Reply:
![image](https://github.com/STARIONGROUP/COMET-WEB-Community-Edition/assets/140734849/ce7b172c-4430-4937-9672-f87072c9c3ba)
Just ran resharper code cleanup to make sure it's ok, but everythings seems fine in VS